packages/edit-post/src/components/device-preview/index.js

```diff
@@ -35,7 +30,7 @@ export default function DevicePreview() {
 			hasActiveMetaboxes: select( editPostStore ).hasMetaBoxes(),
 			isSaving: select( editPostStore ).isSavingMetaBoxes(),
 			isPostSaveable: select( editorStore ).isEditedPostSaveable(),
-			isViewable: get( postType, [ 'viewable' ], false ),
+			isViewable: postType?.viewable,
```

Comment:
All instances in this file work with a potentially nullish `postType`, which is why I've added some optional chaining and nullish coalescing where necessary.

I've removed the default `false` values in a few places where the value is already used as a boolean, because it won't make a difference.

Reply:
Defaulting to `false` still makes sense, because it can avoid unnecessary rerenders. If `getPostType()` call triggers a REST request, then `postType?.viewable` is initially `undefined`, and then becomes either `true` or `false`. The transition from `undefined` to `false` doesn't lead to any UI change, but yet it means a change of `useSelect` return value and triggers a wasted rerender.

Reply:
Point taken, got the `false` instances back in bfdd8424e431add15e6300d600942a1173aaaf69.